js/controllers/list/submissions/SubmissionsListItem.vue

```diff
@@ -212,6 +212,14 @@ export default {
 							notice = this.activeStage.status;
 							break;
 					}
+					if (!this.activeStage.currentUserCanRecommendOnly) {
+						switch (this.activeStage.statusId) {
+							case pkp.const.REVIEW_ROUND_STATUS_RECOMMENDATIONS_READY
+							case pkp.const.REVIEW_ROUND_STATUS_RECOMMENDATIONS_COMPLETED
```

Comment:
Missing the `:` at the end of the `case` statements. This is breaking your builds at the moment.

Reply:
Wow, good catch! :-)